draft-ietf-tls-esni.md

```diff
@@ -621,9 +621,10 @@ variant of ECHClientHello with the following fields:
 - `enc`, as given above; and
 - `payload`, a placeholder byte string containing L zeros.
 
-If optional configuration identifiers (see {{optional-configs}}) are used,
+If random configuration identifiers (see {{random-configs}}) are used,
 `config_id` SHOULD be set to a randomly generated byte in the first
-ClientHelloOuter and MUST be left unchanged for the second ClientHelloOuter.
+ClientHelloOuter and, in the event of HRR, MUST be left unchanged for 
+the second ClientHelloOuter.
```

Comment:
The intent was to say that optional IDs are to be implemented as random IDs, i.e., if you don't care about IDs (hence optional), then just set them to a random byte. So I'm not inclined to take this change.

I _do_ like the clarification to include "in the event of HRR" -- can we split that out into a separate PR?